trin-history/src/validation.rs

```diff
@@ -188,7 +188,8 @@ mod tests {
                         "gasLimit": "0x1c9c364",
                         "gasUsed": "0x140db1",
                         "hash": "0x720704f3aa11c53cf344ea069db95cecb81ad7453c8f276b2a1062979611f09c",
-                        "logsBloom": "", // logs removed since they're unnecessary for validation
+                        // Incorrect logs value used since they're unnecessary for validation
```

Comment:
This wasn't clear to me on first read. Maybe:
```suggestion
                        // Using an empty bloom that doesn't match the real logs, because it is easy and isn't validated
```
Is that what you mean?